PYIC-8660: Update triage pages design for document selection

AmritSidhu opened the pull request:
## Proposed changes
### What changed

- Update triage pages design for document selection

### Why did it change

- Clear upfront guidance on which types of photo ID I can use (and the rules around them)

### Issue tracking
<!-- Jira ticket & other docs, like RFCs -->

- [PYIC-8660](https://govukverify.atlassian.net/browse/PYIC-8660)

## Checklists

- [ ] READMEs and documentation up-to-date
- [ ] Browser/ unit/ Selenium tests have been written/ updated
- [x] No risk of exposure: PII, credentials, etc through logs/ code
- [ ] Ensure added/updated routes have CSRF protection if required


[PYIC-8660]: https://govukverify.atlassian.net/browse/PYIC-8660?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

AmritSidhu committed:
PYIC-8660: Update triage pages design for document selection

undermad commented on locales/cy/translation.json:
```diff
@@ -971,20 +971,40 @@
+        "subHeading": "Pasbort y DU (UK passport)",
+        "paragraph2": "Gallwch ddefnyddio unrhyw basbort y DU os yw heb ddod i ben.",
+        "subHeading2": "Pasbort o’r tu allan i’r DU gyda sglodyn biometrig",
+        "paragraph3": "Gallwch ddefnyddio pasbort o’r tu allan i’r DU os yw: (You can use a non-UK passport if it:)",
```
Here is an English language in Welsh translation. It seems to be also in provided CY content but worth to mention to UCD if that was intended.

AmritSidhu replied:
Confirmed with UCD this was left there by mistake

AmritSidhu committed:
PYIC-8660: Remove english from welsh translations

AmritSidhu committed:
PYIC-8660: Fix details element

AmritSidhu committed:
PYIC-8660: Put sections into a <ul>

This will make it easier for screenreader users to understand the types of ID are related and won’t affect the visual formatting

NoraGDS commented on locales/cy/translation.json:
```diff
@@ -971,20 +971,40 @@
-          "errorSummaryTitleText": "Mae problem",
-          "errorSummaryDescriptionText": "Dewiswch oes os oes gennych unrhyw un o’r mathau hyn o ID gyda llun",
-          "errorRadioMessage": "Dewiswch oes os oes gennych unrhyw un o’r mathau hyn o ID gyda llun"
+          "errorSummaryTitleText": "There is a problem",
```
Change title to Welsh:
`Mae problem`

NoraGDS commented on locales/en/translation.json:
```diff
@@ -903,20 +903,39 @@
+        "paragraph3": "You can use a non-UK passport if it:",
+        "nonUkPassportRequirements": [
+          "has a biometric chip",
+          "has not expired"
```
switch lines 911 and 912:
`has not expired`
`has a biometric chip`

NoraGDS commented on locales/en/translation.json:
```diff
@@ -903,20 +903,39 @@
+          "summaryText": "Has your passport got a biometric chip?",
+          "paragraph4": "To check if your passport has a biometric chip, look for the rectangular biometric chip symbol on the front cover."
+        },
+        "imgAltText": "The front cover of a passport with the biometric chip symbol. The symbol is rectangular with a circle in the middle.",
```
AltText correction:
`The front cover of a passport with the biometric chip symbol enlarged.`

NoraGDS commented on locales/cy/translation.json:
```diff
@@ -971,20 +971,40 @@
+        "paragraph3": "Gallwch ddefnyddio pasbort o’r tu allan i’r DU os yw:",
+        "nonUkPassportRequirements": [
+          "gyda sglodyn biometrig",
+          "heb ddod i ben"
```
switch lines 979 and 980

`heb ddod i ben`
`gyda sglodyn biometrig`

NoraGDS reviewed (commented):
Added comments on content changes

AmritSidhu committed:
PYIC-8660: UCD fixes

Fix spacing
Fix translations
Fix list order
